src/components/Button/index.js

```diff
@@ -1,4 +1,4 @@
-import StyledButton from './styled';
+import StyledButton from './StyledButton';
```

Comment:
As far as I can see you are only using your `StyledButton` as a styled-component (you import it in your form) but you do not use this button component (_.../Button/index.js_), so you can delete it.